addons/link_tracker/models/link_tracker.py

```diff
@@ -84,6 +84,12 @@ def _compute_code(self):
             record = self.env['link.tracker.code'].search([('link_id', '=', tracker.id)], limit=1, order='id DESC')
             tracker.code = record.code
 
+    def _inverse_code(self):
+        self.ensure_one()
+        record = self.env['link.tracker.code'].search([('link_id', '=', self.id)], limit=1, order='id DESC')
+        if record and self.code:
```

Comment:
We could search for a record only if we have self.code (or return if not self.code, that way we avoid an indent level)